tulip/transys/transys.py

```diff
@@ -960,7 +960,11 @@ def simu_abstract(ts, simu_type):
                     lookup[num_cell] = True
                     num_cell += 1
                     if i == j:
+                        Part.remove_edge(num_cell-1, num_cell-1)
                         break
+            if queue.qsize()>1000:
```

Comment:
i kind of don't like hard-coded numbers in the middle of the code. is this part with 1000 crucial?

Reply:
it is part of debuging code, which will be removed later.